File: app/src/main/java/space/pal/sig/repository/dto/NewsPreviewDto.java
package space.pal.sig.repository.dto;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class NewsPreviewDto {

    @SerializedName("name")
    private String name;
    @SerializedName("news_id")
    private String newsId;
    @SerializedName("url")
    private String url;

}

File: app/src/main/java/space/pal/sig/model/dto/NewsPreviewDto.kt
package space.pal.sig.model.dto

import com.google.gson.annotations.SerializedName

/**
 * SpaceNow
 * Created by Catalin on 12/25/2020
 **/
data class NewsPreviewDto(
        @SerializedName("news_id")
        val id: String,
        @SerializedName("name")
        val name: String,
        @SerializedName("url")
        val url: String
)